src/content/docs/style-guide/writing-docs/writer-workflow/github-intro.mdx

```diff
@@ -137,24 +139,24 @@ The Hero currently merges three times a day: At 9 AM (morning), 12 PM (noon), an
 
 To start a release:
 
-1. Create a branch based off develop Github Desktop by clicking **Current Branch** in the top header, clicking **New Branch** in the dropdown, and then selecting **Develop**.
-2. Name the branch following this pattern: `daily-release/mm-dd-yy-morning/noon/evening`. Here's an example: `daily-release/10-27-21-morning`.
-3. Push your changes by clicking **Push Origin** in GitHub Desktop.
+1. Create a branch of `develop` using Github Desktop by clicking **Current Branch**. in the top header, clicking **New Branch** in the dropdown, and then selecting **Develop**.
+2. Name the branch with this pattern: `daily-release/mm-dd-yy-morning/noon/evening`. For example, a daily release happening at 9am on October 27, 2021 would follow this style: `daily-release/10-27-21-morning`.
+3. Push your changes with **Push Origin** in GitHub Desktop.
 4. Create a pull request into main from your new daily release branch by clicking **Create Pull Request**. This will open a pull request screen on [github.com](github.com). Pull requests default to merging into develop, so select main as the `base` branch in the left side of the page and then click `Submit Pull Request`.
 5. Wait until all the checks complete, and then merge the pull request.
 
 
-All branches that follow the `daily-release/mm-dd-yy-morning` pattern are [protected branches](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/about-protected-branches). This means the branches can't be deleted or pushed to by non-admins.
+All branches that follow the `daily-release/mm-dd-yy-morning` pattern are [protected branches](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/about-protected-branches). Only admins can delete them.
 
 ## GitHub labels [#labels]
 
-Every issue needs labels to help us triage and track the health of our backlog:
+We apply labels to issues so we can better triage and track the health of our backlog:
 
-- `content`: Always add, this indicates the issue is content-related rather than a design or engineering issue.
-- `pg_*`: Always add to indicate the product group. For full definitions, see the "Doc Jira and GitHub fields" doc in the internal team Google Drive.
+- `content`: All issues will use this label. `content` indicates the issue is content-related rather than a design or engineering issue.
```

Comment:
```suggestion
- `content`: All issues use this label. `content` indicates the issue is content-related rather than a design or engineering issue.
```

I find that "will" doesn't add much, generally, and can often be left out.